dev/dune-dbg.in

```diff
@@ -45,4 +45,4 @@ while [[ $# -gt 0 ]]; do
     esac
 done
 
-ocamldebug "${opts[@]}" $(ocamlfind query -recursive -i-format coq-core.top_printers) -I +threads -I dev $exe "$@"
+ocamldebug "${opts[@]}" $(ocamlfind query -recursive -i-format coq-core.top_printers coq-core.plugins.extraction) -I +threads -I dev $exe "$@"
```

Comment:
```suggestion
ocamldebug "${opts[@]}" $(ocamlfind query -recursive -i-format coq-core.top_printers) $(ocamlfind query -i-format -descendants coq-core.vernac) -I +threads -I dev $exe "$@"
```
this seems to work to get all plugins

Reply:
Works at least for extraction. Committed. Thanks.